modal/volume.py

```diff
@@ -341,5 +341,17 @@ async def _upload_file(
             mode=file_spec.mode,
         )
 
+    @live_method
+    async def copy_files(self, src_paths: Sequence[Union[str, bytes]], dst_path: Union[str, bytes]) -> None:
+        """
+        Copy files within the volume from src_paths to dst_path.
```

Comment:
As we've found, the behavior of this method's underlying RPC is non-trivial, so I reckon we should provide more info about this method's behavior in the docstring. 

We could simplify things by just including a line like "The semantics of the copy operation follow those of UNIX `cp` command."